examples/run/kokkos/seeding_example_kokkos.cpp

```diff
@@ -123,10 +124,26 @@ int seq_run(const traccc::seeding_input_config& i_cfg,
                     common_opts.input_data_format, &host_mr);
             }  // stop measuring hit reading timer
 
+            {
+                traccc::performance::timer t("Alt hit reading  (cpu)",
+                                             elapsedTimes);
+                // Read the hits from the relevant event file
+                alt_spacepoints_per_event = traccc::io::read_spacepoints_alt(
+                    event, common_opts.input_directory, surface_transforms,
+                    common_opts.input_data_format, &host_mr);
+            }  // stop measuring hit reading timer
+
+            std::cout << "here " << __LINE__ << std::endl;
+
             {  // Spacepoin binning for kokkos
-                m_spacepoint_binning(traccc::get_data(spacepoints_per_event));
+                traccc::performance::timer t("Spacepoint binning (kokkos)",
+                                             elapsedTimes);
+                m_spacepoint_binning(
+                    vecmem::get_data(alt_spacepoints_per_event));
             }
 
+            std::cout << "here " << __LINE__ << std::endl;
```

Comment:
I am guessing this is a debugging leftover. :slightly_smiling_face:

Reply:
whoops